reference/docs-conceptual/getting-started/cookbooks/Decode-PowerShell-Command-from-a-Running-Process.md

```diff
@@ -0,0 +1,92 @@
+# Decode PowerShell Command from a Running Process
+
+There are times when I've found a PowerShell process running that is taking up a bunch of resources. Sometimes they've even been my own scripts running in the context of a [Task Scheduler](https://docs.microsoft.com/windows/desktop/TaskSchd/task-scheduler-start-page
+) job or a [SQL Server Agent](https://docs.microsoft.com/sql/ssms/agent/sql-server-agent
+) Job with a PowerShell job step. Because multiple PowerShell processes can be running at a time, sometimes it's difficult to know what the offending process is. The following method can be used to decode a script block that a PowerShell process is currently running.
+
+## Create a Long Running Process
+
+To demonstrate this capability, open a new PowerShell window and run the following code. It executes a PowerShell command that outputs a number every minute for 10 minutes.
+
+```PowerShell
+powershell.exe -Command {
+    $i = 1
+    while ( $i -le 10 )
+    {
+        Write-Output -InputObject $i
+        Start-Sleep -Seconds 60
+        $i++
+    }
+}
+```
+
+## View the Process
+
+### Using Task Manager
+
+- Launch the Task Manager and select the **Details** tab.
+- Scroll down the list to powershell.exe.
+
+  [![Task Manager Details](./media/DecodeCommandTaskManagerDetails.png)](./media/DecodeCommandTaskManagerDetails.png "Task Manager Details")
+
+- Right-click the column headers in Task Manager and click **Select Columns**.
+
+  [![Task Manager right-click the column header](./media/DecodeCommandTaskManagerRightClick.png)](./media/DecodeCommandTaskManagerRightClick.png "Right-Click the column header")
+
+- Check **Command Line** in the Select columns dialogue and click **OK**.
+
+  [![Task Manager select **Columns**](./media/DecodeCommandTaskManagerSelectColumns.png)](./media/DecodeCommandTaskManagerSelectColumns.png "Select Columns")
+
+- Now the parameters that were passed into powershell.exe are visible. However, the command is still obfuscated as an encoded command.
+
+  [![Task Manager with Command Line](./media/DecodeCommandTaskManagerWithCommandLine.png)](./media/DecodeCommandTaskManagerWithCommandLine.png "Task Manager with Command Line shown")
+
+### Using PowerShell
+
+- Start PowerShell as Administrator. It is vital that PowerShell is running as administrator, otherwise no results will be returned when querying the running processes.
```

Comment:
This does not need to be a bulleted list. Just use paragraphs.

You are diving right into the steps without explaining the purpose. Explain that you want to find instances of PowerShell that are using the **EncodedCommand** parameter.

Reply:
Done.